.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
         # Only needed for private caches
         #authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+    # This might be overkill: telomare cache could be enough.
     - run: cachix use iohk
```

Comment:
Have you tried changing to telomare and running it?

Reply:
do you mean changing line 21 to: `- run: cachix use iohk` ?

If yes, this is already achieved by:
```
     - uses: cachix/cachix-action@v6
      with:
        name: telomare
        signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
```
7 lines before

All of this if I'm right... many times I haven't been.